Traceback (most recent call last):
  File "genSatProps.py", line 46, in <module>
RuntimeError: input(): lost sys.stdin
Traceback (most recent call last):
  File "genSatProps.py", line 46, in <module>
RuntimeError: input(): lost sys.stdin
Traceback (most recent call last):
  File "genSatProps.py", line 46, in <module>
RuntimeError: input(): lost sys.stdin
Traceback (most recent call last):
  File "genSatProps.py", line 46, in <module>
RuntimeError: input(): lost sys.stdin
